### PR TITLE
[tidb] Add tidb cdc connector e2e test.

### PR DESCRIPTION
**Pull request by GOODBOY008**
Issue https://github.com/ververica/flink-cdc-connectors/issues/932. Please just review lastest commit.

### CODE REVIEW

**Comment by leonardBang** [comment]
@GOODBOY008 Could you rebase to latest master ?

**Comment by GOODBOY008** [comment]
> 



> @GOODBOY008 Could you rebase to latest master ?

Ok

**Review by leonardBang — COMMENTED** [review]
Thanks @GOODBOY008 for the contribution, LGTM. The CI failure which may same with 
#875

File: flink-cdc-e2e-tests/src/test/java/com/ververica/cdc/connectors/tests/TIDBE2eITCase.java [hunk 1]
```diff
@@ -0,0 +1,274 @@
...
+                        "106,hammer,18oz carpenter hammer,1.0",
+                        "107,rocks,box of assorted rocks,5.1",
+                        "108,jacket,water resistent black wind breaker,0.1",
+                        "109,spare tire,24 inch spare tire,22.2",
+                        "110,jacket,new water resistent white wind breaker,0.5");
+        proxy.checkResultWithTimeout(
+                expectResult,
+                "products_sink",
+                new String[] {"id", "name", "description", "weight"},
+                360000L);
```
Comment (leonardBang) [inline flink-cdc-e2e-tests/src/test/java/com/ververica/cdc/connectors/tests/TIDBE2eITCase.java:228]: Could you also update  the time out for postgres and others e2e test?

**Review by leonardBang — APPROVED** [review]
Thanks @GOODBOY008 for the continuous effort for the unstable tests, LGTM!

**Pull request merged**